### PR TITLE
Dynamic searching for images of user-provided type

### PR DESCRIPTION
**Pull request by Graham-Foster**
Added attribute 'image-topic' 

This will be the search term for a Google Images search and the first few results will be used for the carosel.

Because the search takes a little while, the user should still manually provide a few images to be used while the others load.

### CODE REVIEW

File: fancy-carousel.html [hunk 1]
```diff
@@ -88,12 +94,39 @@
...
           this.$.prevBtn.style.display = 'none';
           this.$.nextBtn.style.display = 'none';
         }
+        if (this.imageTopic) {
+          this._loadCustomImages(this.imageTopic);
+        }
+      },
+
+      _loadCustomImages: function(searchTerm) {
+        alert('searching for searchTerm: ' + searchTerm);
```
Comment (annsonn) [inline fancy-carousel.html]: Why is this here?!

Comment (Graham-Foster) [reply fancy-carousel.html]: my bad...
